docs/cephfs-snap-clone.md

```diff
@@ -0,0 +1,255 @@
+# Create CephFS snapshot and Clone Volume
+
+- [Create CephFS snapshot and Clone Volume](#create-cephfs-snapshot-and-clone-volume)
+  - [Prerequisite](#prerequisite)
+    - [Snapshot PVCs](#snapshot-pvcs)
+  - [create snapshotclass](#create-snapshotclass)
+  - [create snapshot](#create-snapshot)
+  - [Get details about the snapshot](#get-details-about-the-snapshot)
+  - [Get details about the volumesnapshotcontent](#get-details-about-the-volumesnapshotcontent)
+  - [Restore snapshot to a new PVC :](#restore-snapshot-to-a-new-pvc-)
+    - [CLONE PVCs](#clone-pvcs)
+  - [Get details about the cloned PVC](#get-details-about-the-cloned-pvc)
+
+## Prerequisite
+
+- For snapshot functionality to be supported for your kubernetes cluster, the
+  kubernetes version running in your cluster should be >= v1.17. We also need the
+  `snapshot controller` deployed in your kubernetes cluster along with `csi-snapshotter`
+  sidecar container. These sidecar controllers watch for snasphot requests and push the
+  requests to Ceph CSI driver for snapshot creation,deletion operations. There should
+  be a `volumesnapshotclass` object present in the cluster for snapshot request to be satisified.
+
+
+### Snapshot PVCs
+
+```bash
+## create snapshotclass
+$kubectl create -f snapshotclass.yaml
+
+## create snapshot
+
+$ kubectl create -f snapshot.yaml
```

Comment:
please add a link to cephfs readme for how to create PVC and also mention that PVC should be already created to create a snapshot

Reply:
done.